### ISSUE TITLE
Bug in possible values endpoint?

### ISSUE THREAD

**Issue by JoelRichardson** [intermine/intermine#1725]
I think there may be a bug in the path/values endpoint where it doesn't look at the whole path.
To reproduce:

Go here: http://iodocs.apps.intermine.org/mgi/docs#/ws-possible-values/GET/path/values
Enter the path "Allele.feature.organism.taxonId"

You will get back all the organisms in the Organisms class instead of just ones accessible along this path, which should return only mouse. If you do the equivalent query in the query builder, you get the correct answer.

**Comment by julie-sullivan** [comment]
The code is only retrieving the last class and field:
https://github.com/intermine/intermine/blob/dev/intermine/web/main/src/org/intermine/webservice/server/path/PossibleValuesService.java#L152-L153

So in your example, it'd be getting the values for `organism.taxonId` only, as you are seeing. It's discarding the rest of the path.

**Comment by JoelRichardson** [comment]
Is that the intended behavior? The document string for the service just says: This service provides the possible values that a path may represent in the database. This functionality is expected to primarily useful for applications providing completion and suggestions for user-input.

**Comment by julie-sullivan** [comment]
Yes, it should be suggesting values for the full path, not just the last class.

I suppose to fix you would cycle through the path and add constraints to the query. Instead of just querying for the end class and field.
